README.md

```diff
@@ -67,6 +67,10 @@ brew autoupdate version:
                                    instead of waiting for one interval (24
                                    hours by default) to pass first. Must be
                                    passed with start.
+      --sudo                       If a Cask requires sudo, autoupdate will 
+                                   open a GUI to ask for the password.
+                                   Requires https://formulae.brew.sh/formula/pinentry-mac
+                                   to be installed.
```

Comment:
It would be better to use `Formula["pinentry-mac"].any_version_installed?` in the code rather than documenting the requirement.

Reply:
Ok, I will add a check in the code.

Reply:
I added it in https://github.com/Homebrew/homebrew-autoupdate/pull/110/commits/83b26e9e46fe5ff1e5bc1c9d5d682945a5d0a827